include/grpcpp/impl/codegen/callback_common.h

```diff
@@ -85,6 +85,10 @@ class CallbackWithStatusTag
       : call_(call), func_(std::move(f)), ops_(ops) {
     g_core_codegen_interface->grpc_call_ref(call);
     functor_run = &CallbackWithStatusTag::StaticRun;
+    // A client-side callback should never be run inline since they will always
```

Comment:
non-blocking comment - never sounds like a strong word.. what if it's a fast no IO function?